website/public/js/controllers/notificationCtrl.js

```diff
@@ -53,6 +60,19 @@ habitrpg.controller('NotificationCtrl',
        Notification.mp(mana);
     });
 
+    $rootScope.$watch('user.stats.lvl', function(after, before) {
+      if (after === before) return; 
```

Comment:
Ah, sure. That'd mean we can remove an extra `if` at line 69, as well. I thought I recalled watches acting weird if you didn't have a strict-equality check at the beginning, but maybe we're safe when we're working with a numeric attribute no user will ever lack.